### PR TITLE
Session start issue

### PR DESCRIPTION
**Pull request by BrandonStalnaker**
# Summary

In our Livestream, the regular events event have the same session_id and session_uuid but Session Start has a different one which is incorrect.

### CODE REVIEW

**Review by alexs-mparticle — APPROVED** [review]

**Comment by alexs-mparticle** [comment]
LGTM, just update the Summary to explain the history and difference between `sid` and `id` for posterity.

File: mParticleCore.brs [hunk 1]
```diff
@@ -1,74 +1,74 @@
 '*************************************************************
 ' mParticle Roku SDK - Core APIs
-' Copyright 2019 mParticle, Inc. 
+' Copyright 2019 mParticle, Inc. test
```
Comment (samdozor) [inline mParticleCore.brs:3]: oops ^^

**Review by samdozor — CHANGES_REQUESTED** [review]
- I'd like to make sure that the formatting change ends up in the history as a separate commit, so that in the future on a `git blame`, these lines are shown as changing for the right reasons. Can you separate the style change into a separate PR?

- Please also re-write your commit messages to be more consistent with the commits of the roku repo:

Screen Shot 2020-10-13 at 8 48 54 AM

"Formatting for change to using VSCode" <-- commits should not refer to themselves in the 3rd person, and the "to using" is awkward grammar. And who is using vscode? I took a look at the vscode plugin, and actually it's [using this project under the hood](https://github.com/RokuCommunity/brighterscript-formatter). So it's more accurate to describe this as something like "Reformat code with brighterscript-formatter"

"The id of the session start event is the sid" <-- commit shouldn't state fact like this. This would be as if we had other commits such as "The SDK version is 1.2". How about "Use the session ID as the session start message ID" - and then you can include a longer description on the 3rd line of the commit message describing why this change was needed.

**Comment by BrandonStalnaker** [comment]
@samdozor Yeah I was writing those more explain to each other and was planning to write a better commit message with the merge commit. You're right though. The commits should be kept separate. Here's the formatting commit. https://github.com/mParticle/mparticle-roku-sdk/pull/13

**Comment by BrandonStalnaker** [comment]
Made a new PR https://github.com/mParticle/mparticle-roku-sdk/pull/14

**Pull request closed without merging**